graphql/e2e/auth/private_key.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Why will we put the private key in code? It defeats any purpose of using RSA. 
Also, are you using RSA 1024 bit or 2048 bit? Best to use 2048, 4096 bits.